packages/dcos-diagnostics/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-diagnostics.git",
-    "ref": "096c5771987e2f489c9ba0369b080f1bb99bce03",
+    "ref": "fb14433a6c58fc6fe9ac3570b4a33ea47d909f0d",
```

Comment:
You'll need to bump the Go version of pkgpanda's builder Docker image like I did here for the 1.11 branch: https://github.com/dcos/dcos/commit/f7e0c5973f945dd41041c5460ffce6cfe952cd56